routers/api/v1/convert/convert.go

```diff
@@ -190,3 +190,14 @@ func ToTeam(team *models.Team) *api.Team {
 		Permission:  team.Authorize.String(),
 	}
 }
+
+// ToTrackedTime convert models.TrackedTime to api.TrackedTime
+func ToTrackedTime(trackedTime *models.TrackedTime) *api.TrackedTime {
```

Comment:
I think this better would be as model method APIFormat() like this: https://github.com/go-gitea/gitea/blob/aa962deec037375a7cfae3b96094becdfe48e778/models/issue.go#L268

Reply:
@lafriks Done. Out of curiosity, why do you prefer a `TrackedTime.APIFormat()` method to a `convert.ToTrackedTeam` function?

Reply:
@ethantkoenig it's just more straightforward and when just converting model to api type (1:1) I think there is no need for special function